src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -1,55 +1,125 @@
-import React, { useCallback, useState } from "react";
+import React, { useCallback } from "react";
 import { Modal } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+import { MAX_TABLE_CAPACITY, MIN_TABLE_CAPACITY } from "settings";
+
+import { InputField } from "components/atoms/InputField";
 
 import "./EditTableTitleModal.scss";
 
 export interface EditTableTitleModalProps {
   isShown: boolean;
   title: string;
+  description?: string;
```

Comment:
Subtitle?

Reply:
Check this: https://github.com/sparkletown/sparkle/pull/1377#discussion_r632474494